components/service-operator/apis/database/v1beta1/postgres_types.go

```diff
@@ -166,6 +169,13 @@ func (p *Postgres) GetStackTemplate() (*cloudformation.Template, error) {
 		BackupRetentionPeriod: DefaultBackupRetentionPeriod,
 	}
 
+	// for backwards compatibility only specify EngineVersion if the k8s object specifies it
+	if len(p.Spec.AWS.EngineVersion) > 0 {
```

Comment:
Now that I think about it...this if statement actually isn't necessary. Inlining it will probably do The Right Thing:tm:. Alas, my AWS access has been revoked already so I can't run the tests to confirm it.